readthedocs/subscriptions/products.py

```diff
@@ -134,7 +134,7 @@ def get_feature(obj, feature_type) -> RTDProductFeature:
     available_stripe_products_id = [
         product.stripe_id for product in get_products_with_feature(feature_type)
     ]
-    stripe_subscription = organization.get_or_create_stripe_subscription()
+    stripe_subscription = organization.stripe_subscription
```

Comment:
This was trying to create a subscription during tests, previously this wasn't a problem because if the Plan model didn't exist, we skipped the creation of the subscription. Now all organizations have a subscription attached, so we don't need to use get or create here.